qa-bloodpac.planx-pla.net/manifest.json

```diff
@@ -79,47 +150,172 @@
+          {
+            "name": "sower-jobs-creds-volume",
+            "readOnly": true,
+            "mountPath": "/creds.json",
+            "subPath": "creds.json"
+          }
+        ],
+        "cpu-limit": "1",
+        "memory-limit": "1Gi"
+      },
+      "volumes": [
+        {
+          "name": "sower-jobs-creds-volume",
+          "secret": {
+            "secretName": "sower-jobs-g3auto"
+          }
+        }
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "indexd-manifest",
+      "action": "download-indexd-manifest",
+      "activeDeadlineSeconds": 86400,
+      "serviceAccountName": "jobs-data-bloodpac-org",
+      "container": {
+        "name": "job-task",
+        "image": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/download-indexd-manifest:2022.08",
+        "pull_policy": "Always",
+        "env": [
+          {
+            "name": "GEN3_HOSTNAME",
+            "valueFrom": {
+              "configMapKeyRef": {
+                "name": "manifest-global",
+                "key": "hostname"
+              }
+            }
+          }
+        ],
+        "volumeMounts": [
+          {
+            "name": "sower-jobs-creds-volume",
+            "readOnly": true,
+            "mountPath": "/creds.json",
+            "subPath": "creds.json"
+          }
+        ],
+        "cpu-limit": "1",
+        "memory-limit": "1Gi"
+      },
+      "volumes": [
+        {
+          "name": "sower-jobs-creds-volume",
+          "secret": {
+            "secretName": "sower-jobs-g3auto"
+          }
+        }
+      ],
+      "restart_policy": "Never"
     }
   ],
+  "jupyterhub": {
+    "enabled": "yes",
+    "nb_storage_size": "250Gi",
+    "sidecar": "quay.io/cdis/gen3fuse-sidecar:2022.05",
+    "containers": [
+      {
+        "name": "Bioinfo - Python/R",
+        "cpu": 0.5,
+        "memory": "256M",
+        "image": "quay.io/occ_data/jupyternotebook:1.7.4"
+      },
+      {
+        "name": "Bioinfo - Python/R",
+        "cpu": 1,
+        "memory": "1024M",
+        "image": "quay.io/occ_data/jupyternotebook:1.7.4"
+      },
+      {
+        "name": "Bioinfo - Python/R",
+        "cpu": 6,
+        "memory": "15512M",
+        "image": "quay.io/occ_data/jupyternotebook:1.7.4"
+      },
+      {
+        "name": "Bioinfo - Python/R-3.6",
+        "cpu": 0.5,
+        "memory": "256M",
+        "image": "quay.io/occ_data/jupyternotebook:1.8.0"
+      },
+      {
+        "name": "Bioinfo - Python/R-3.6",
+        "cpu": 1,
+        "memory": "1024M",
+        "image": "quay.io/occ_data/jupyternotebook:1.8.0"
+      },
+      {
+        "name": "Bioinfo - Python/R-3.6",
+        "cpu": 6,
+        "memory": "15512M",
+        "image": "quay.io/occ_data/jupyternotebook:1.8.0"
+      }
+    ]
+  },
   "peregrine": {
     "sidecar": "True"
   },
   "guppy": {
     "indices": [
       {
-        "index": "qa-bloodpac.planx-pla.net_study",
+        "index": "bloodpac_study",
```

Comment:
the indices can't change, the names are set here https://github.com/uc-cdis/gitops-qa/blob/master/qa-bloodpac.planx-pla.net/etlMapping.yaml